docs/website/blog/2022-05-02-odo-v3-alpha1.md

```diff
@@ -0,0 +1,319 @@
+
+## Introducing odo v3
+
+You can find the instructions to install odo v3-alpha1 from [this release page](https://github.com/redhat-developer/odo/releases/tag/v3.0.0-alpha1). The binaries are accessible [here](https://developers.redhat.com/content-gateway/rest/mirror/pub/openshift-v4/clients/odo/v3.0.0~alpha1/).
+
+### Initializing a project
+
+The `odo init` command is the first command to use, before to start using `odo` with your project. The goal of this first step is to get a suitable devfile for your project.
+
+`odo init` will search for devfiles into devfile registries. By default, `odo` is configured to access only one devfile registry (the one specified above), and you can modify the devfile registries `odo` is accessing using the command `odo preference registry`.
+
+This `odo init` command offers two modes, either interactive, or manual. The interactive mode will help you discover the appropriate devfile. To use the interactive mode, you just need to enter `odo init` in your command line.
+
+If you execute this command from a directory containing sources, `odo` will try to recognize the language and framework you are using, will search into the devfile registries you have configured the most appropriate devfile, and give you the choice to use this one, or to search for another one. 
+
+```
+$ odo init
+  __
+ /  \__     Initializing new component
+ \__/  \    Files: Source code detected, a Devfile will be determined based upon source code autodetection
+ /  \__/    odo version: v3.0.0-alpha1
+ \__/
+
+Interactive mode enabled, please answer the following questions:
+Based on the files in the current directory odo detected
+Language: javascript
+Project type: nodejs
+The devfile "nodejs" from the registry "DefaultDevfileRegistry" will be downloaded.
+? Is this correct? (Y/n) 
+```
+
+If you answer `No` here, or if you run the `odo init` command from an empty directory, `odo init` will help you choose the appropriate devfile. The command will also help you make some personalization on the devfile, by personalizing the endpoints and the environment variables for the container that will be deployed during the development phase.
+
+```
+? Select language: javascript
+? Select project type: Node.js Runtime
+ ✓  Downloading devfile "nodejs" from registry "DefaultDevfileRegistry" [961ms]
+Current component configuration:
+Container "runtime":
+  Opened ports:
+   - 3000
+  Environment variables:
+? Select container for which you want to change configuration? NONE - configuration is correct
+? Enter component name: my-nodejs-app
+
+Your new component 'my-nodejs-app' is ready in the current directory.
+To start editing your component, use 'odo dev' and open this folder in your favorite IDE.
+Changes will be directly reflected on the cluster.
+```
+
+Finally, if you start the `odo init` command from an empty directory, it will give you the choice to download one of the starter projects listed in the devfile.
```

Comment:
```suggestion
Finally, if you start the `odo init` command from an empty directory, it will give you the choice to download one of the starter projects listed in the Devfile.
```